src/NotesLoaderSM.cpp

```diff
@@ -979,26 +979,57 @@ bool SMLoader::LoadFromBGChangesString( BackgroundChange &change, const RString
 		// Backward compatibility:
 		if( change.m_def.m_sEffect.empty() )
 		{
-			bool bLoop = std::stoi( aBGChangeValues[5] ) != 0;
-			if( !bLoop )
```

Comment:
This change inverts this condition, as per the spec "6. Backward compatible effect flag. StretchNoLoop is used if this is not 0.", see: https://github.com/stepmania/stepmania/wiki/sm#bgchanges

Reply:
This means, this implementation is wrong, assuming the spec is correct.